stable/datadog/templates/container-system-probe.yaml

```diff
@@ -1,6 +1,10 @@
 {{- define "container-system-probe" -}}
 - name: system-probe
+  {{- if semverCompare "^6.15.x" .Values.image.tag }}
   image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+  {{- else }}
+  image: "{{ .Values.image.repository }}:6.15.0"
+  {{- end }}
```

Comment:
The default version in the values.yaml is now "6" that is greater than 6.15, so you can remove this change.